sdk/communication/azure-communication-common/src/main/java/com/azure/communication/common/CommunicationIdentifier.java

```diff
@@ -9,6 +9,81 @@
  */
 public abstract class CommunicationIdentifier {
 
+    /**
+     * Prefix for a phone number.
+     */
+    protected static final String PHONE_NUMBER = "4:";
+
+    /**
+     * Prefix for a bot.
+     */
+    protected static final String BOT = "28:";
+
```

Comment:
Just a recommendation  :
Rename this constant as "BOT_GLOBAL" since it is used when isGlobal flag is true

Reply:
Thanks, updated

Reply:
maybe we should it revert it back, as here we are mentioning only the first part of the MRI and as 28 identifies all global and non global bot identifiers, wdyt @mjafferi-msft @ostoliarova-msft ? :)

Reply:
The reason for adding GLOBAL Prefix was that we are setting isGlobal flag as true when this constant was compared (Line 106 & 107). So, it means it just identifies global bot here.

Reply:
> The reason for adding GLOBAL Prefix was that we are setting isGlobal flag as true when this constant was compared (Line 106 & 107).

No, the reason we're setting that `isGlobal` flag is that we're comparing `segments.length == 2` (that's what makes it different from tenantized bots).

Please stick to semantics, if ever necessary, you can even introduce two constants with the same value `28:` with different names used in different contexts.

Reply:
@mjafferi-msft, @AikoBB please advice on what the naming should be according to Java best practices

Reply:
in this case the constant should answer the question `what it is` not `why we introduced it and where it's used`, so I vote for reverting back to BOT which explains what is this prefix as all bot identifier prefixes start with `28:` and it will be consistent with other platforms as well

Reply:
agreed

Reply:
updated